spec/datetimeformat.html

```diff
@@ -80,7 +80,7 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. If _hour12_ is *true*, then
           1. If _hcDefault_ is *"h11"* or *"h23"*, let _hc_ be *"h11"*. Otherwise, let _hc_ be *"h12"*.
         1. Else if _hour12_ is *false*, then
-          1. If _hcDefault_ is *"h11"* or *"h23"*, let _hc_ be *"h23"*. Otherwise, let _hc_ be *"h24"*.
+          1. Let _hc_ be *"h23"*.
```

Comment:
Why are you changing it here but not 2 lines above?

Reply:
because if I change the 2 line above, then ja will not work
see https://github.com/tc39/ecma402/issues/402#issuecomment-623628320
see the following code
```js
(new Intl.DateTimeFormat("ja", {hour: "numeric", hour12: true})).resolvedOptions().hourCycle;
// h11
(new Intl.DateTimeFormat("en", {hour: "numeric", hour12: true})).resolvedOptions().hourCycle
// h12
(new Intl.DateTimeFormat("fr", {hour: "numeric", hour12: true})).resolvedOptions().hourCycle
// h12
```
and this should be kept.